Sources/SwiftSDKGenerator/Serialization/SwiftSDKMetadata.swift

```diff
@@ -73,8 +73,8 @@ struct DestinationV3: Encodable {
 }
 
 /// Represents v4 schema of `swift-sdk.json` (previously `destination.json`) files used for cross-compilation.
-struct SwiftSDKMetadataV4: Encodable {
-  struct TripleProperties: Encodable {
+public struct SwiftSDKMetadataV4: Encodable {
+  public struct TripleProperties: Encodable {
```

Comment:
Let's avoid `public` in new code and make all APIs explicitly local to this package.

```suggestion
package struct SwiftSDKMetadataV4: Encodable {
  package struct TripleProperties: Encodable {
```